pom.xml

```diff
@@ -99,7 +99,7 @@ THE SOFTWARE.
     <changelog.url>https://jenkins.io/changelog</changelog.url>
 
     <!-- Bundled Remoting version -->
-    <remoting.version>4.6</remoting.version>
+    <remoting.version>4.7-rc2860.f4a9a135655d</remoting.version>
```

Comment:
This doesn't yet exist publicly. Looks like there are currently infra issues preventing it from being generated publicly.